components/Tabs/tests/unit/MDCTabBarControllerEncodingTests.m

```diff
@@ -55,34 +55,38 @@ @interface MDCTabBarControllerEncodingTests : XCTestCase
 @implementation MDCTabBarControllerEncodingTests
 
 //TODO: (#3220) [TabBar] Re-enable test that is throwing an exception
-//- (void)testTabBarControllerEncoding {
-//  MDCTabBarViewController *tabBarVC = [[MDCTabBarViewController alloc] init];
-//  [tabBarVC.view setBackgroundColor:[UIColor clearColor]];
-//
-//  UIViewController *vc1 = [[UIViewController alloc] init];
-//  UIViewController *vc2 = [[UIViewController alloc] init];
-//  vc1.view.tag = 1;
-//  vc2.view.tag = 2;
-//  tabBarVC.viewControllers = @[vc1, vc2];
-//  tabBarVC.selectedViewController = vc1;
-//  tabBarVC.tabBar.tintColor = [UIColor colorWithRed:0.6f green:0.2f blue:0.3f alpha:1.0f];
-//  MDCTabBarViewControllerDelegateExample *delegate = [[MDCTabBarViewControllerDelegateExample alloc] init];
-//  tabBarVC.delegate = delegate;
-//  // force view did load
-//
-//  NSData *archivedData = [NSKeyedArchiver archivedDataWithRootObject:tabBarVC];
-//  MDCTabBarViewController *unarchivedTabBarVC =
-//      [NSKeyedUnarchiver unarchiveObjectWithData:archivedData];
-//  XCTAssertEqual(unarchivedTabBarVC.view.subviews.count, tabBarVC.view.subviews.count);
-//  XCTAssertEqual(unarchivedTabBarVC.viewControllers[0].view.tag,
-//                 tabBarVC.viewControllers[0].view.tag);
-//  XCTAssertEqual(unarchivedTabBarVC.viewControllers[1].view.tag,
-//                 tabBarVC.viewControllers[1].view.tag);
-//  XCTAssertEqual(unarchivedTabBarVC.selectedViewController.view.tag,
-//                 tabBarVC.selectedViewController.view.tag);
-//  XCTAssertEqualObjects(unarchivedTabBarVC.tabBar.tintColor, tabBarVC.tabBar.tintColor);
-//  XCTAssertNil(unarchivedTabBarVC.delegate);
-//}
+- (void)_disabled_testTabBarControllerEncoding {
```

Comment:
please make the re-enabling of the test as a separate PR

Reply:
The test remains disabled.  clang-format was making me angry by suggesting stupid reformattings of the commented code.  I decided it was simpler to uncomment it than it was to fight the formatter (or make changes that would break the code when uncommented.